src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -141,7 +149,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
 
             if (!$fs->exists($warmupDir.'/'.$containerDir)) {
-                $fs->rename($realCacheDir.'/'.$containerDir, $warmupDir.'/'.$containerDir);
+                $fs->rename(($useBuildDir ? $realBuildDir : $realCacheDir).'/'.$containerDir, $warmupDir.'/'.$containerDir);
```

Comment:
I think `$realBuildDir` is enough as it equals `$realCacheDir`

Reply:
Yes, you are correct. 

Thank you